tests/pds/peppi/test_products.py

```diff
@@ -1,10 +1,13 @@
+import logging
 import unittest
 from datetime import datetime
 from typing import get_args
 
 import pds.peppi as pep  # type: ignore
 from pds.api_client import PdsProduct
 
+logger = logging.getLogger(__name__)
```

Comment:
Why was this added? I don't see any logging going on in this test suite now.

Reply:
I can remove that, I think I added when I was unable to assert something in the logs, I need to check that this works without that.